RecoTracker/LSTCore/interface/LSTESData.h

```diff
@@ -21,7 +21,8 @@ namespace lst {
     unsigned int nEndCapMap;
     std::unique_ptr<const PortableMultiCollection<TDev, ModulesSoA, ModulesPixelSoA>> modules;
     std::unique_ptr<const PortableCollection<EndcapGeometryDevSoA, TDev>> endcapGeometry;
-    std::shared_ptr<const PixelMap> pixelMapping;
+    std::shared_ptr<const PixelMap>
+        pixelMapping;  // Host-side object that is shared between the LSTESData<TDev> objects for different devices
```

Comment:
Can you put this comment on a separate line just above to avoid the awkward line break?

Reply:
Yeah, sounds good